inc/class-base-css.php

```diff
@@ -292,12 +327,14 @@ public function cycle_through_static_blocks( $blocks ) {
 				$path = new $classname();
 
 				if ( method_exists( $path, 'render_css' ) ) {
-					if ( $this->library_prefix . '/' . $path->block_prefix === $block['blockName'] ) {
+					if ( ( isset( $path->library_prefix ) ? $path->library_prefix : $this->library_prefix ) . '/' . $path->block_prefix === $block['blockName'] ) {
 						$style .= $path->render_css( $block );
 					}
 				}
 			}
 
+			$style .= apply_filters( 'themeisle_gutenberg_blocks_css', $block );
```

Comment:
@HardeepAsrani, any idea on where this function is located? It does not seams to exists in the Gutenberg CSS repo.

Reply:
It exists on Gutenebrg CSS repo, you can find it here: https://github.com/Codeinwp/gutenberg-css/pull/308/files#diff-701c56341bae3e4388c24f0344860502d56c3c6898a2eccf1d94c75c24af0882R40

Will be great if you can find it in a manner where I can reproduce it. I tried with the code you sent but it doesn't work - but I can confirm that the same error once happened to me randomly but never happened again.

Reply:
I see.